src/billing/utils/timeHelpers.ts

```diff
@@ -58,3 +60,19 @@ export const minToSeconds = (min: number) => {
   }
   return min * secondsPerMinute
 }
+
+export const nsToSeconds = (ns: number): number => {
+  if (ns === -1) {
+    return ns
+  }
+
+  return ns / nanoPerSecond
+}
+
+export const secondsToNs = (seconds: number): number => {
+  if (seconds === -1) {
+    return seconds
+  }
+
+  return seconds * nanoPerSecond
+}
```

Comment:
I'd love to see some unit tests added for these two methods. Since there are two exit conditions for each method (when the input is `-1` and when the input is any other number) there should ideally be four tests or four assertions.

Reply:
I completely missed that there is a separate test file specifically for the timeHelpers. Will add those.